src/details/MovieDetailsCard.js

```diff
@@ -23,13 +24,15 @@ class MovieDetailsCard extends Component {
       return (
         <div className='movie-details-card' style={{backgroundImage: `url(${this.state.movie.backdrop_path})`}}>
          <div className='movie-details-text'>
-           <button className='close-button' onClick={() => this.state.displayMovieLibrary()}>X</button>
+          <Link to={`/`}>
+            <button className='close-button'>X</button>
+          </Link>
            <h1 className='movie-title'>{this.state.movie.title}</h1>
```

Comment:
I think moving the className='close-button' up to the Link element will solve the styling issue we're seeing here. Might impact the tests though...